internal/olm/client/client_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	olmapiv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	fake "sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var _ = Describe("check depErrors", func() {
+	Describe("create k8s objects", func() {
```

Comment:
I like something joe suggested to me once, making the test look like the godoc for the object would. So I'd expect to see the following:

```golang
var _ = Describe("Client", func() {
	Describe("printDeploymentErrors", func() {
		var (...)
		BeforeEach(....)
		Context("with a valid csv", func() {
			It("should validate the csv successfully", func() {
				...
			})
		})
	})
})
```

If you run `gingko -v` you can actually see the printout and it should read almost like a sentence.